src/documents/CoreAPI.md

```diff
@@ -0,0 +1,635 @@
+---
+title: Core API
+layout: default
+category: Infusion
+---
+
+# Infusion Core API
+
+This file documents some low-level APIs which are provided by Infusion at the JavaScript level. Note that since
```

Comment:
There is an awful lot of info on this page (which is great!). It might be helpful to add a table of contents of at least the sections, so that readers have a sense of what's here, and to help them find what they're probably looking for.